packages/ide/src/fill/electron.ts

```diff
@@ -47,7 +47,8 @@ const newCreateElement = <K extends keyof HTMLElementTagNameMap>(tagName: K): HT
 			},
 			set: (value: string): void => {
 				if (value) {
-					value = value.replace(/file:\/\//g, "/resource");
+					const resourceBaseUrl = location.pathname.replace(/\/$/, "") + "/resource";
```

Comment:
Wouldn't this only catch URLs that end with `/`?

Something like `https://ide.kwc.io/banana` wouldn't be caught with this.

Reply:
This makes sure the url is correct.

Both 
`https://ide.kwc.io/banana`
`https://ide.kwc.io/banana/`
use
`https://ide.kwc.io/banana/resource`
to request the resources